fabric-resource-loader-v0/src/main/resources/assets/fabric-resource-loader-v0/lang/en_us.json

```diff
@@ -1,4 +1,5 @@
 {
+  "pack.fabricapi.description": "Mod resources.",
```

Comment:
Shouldn't this translation key be `pack.description.fabricMod` or similar to better match the other translation keys?

Reply:
Perhaps this would make more sense